FileSystem/Util.js

```diff
@@ -38,3 +38,96 @@ function deleteFile(name) {
     DriveApp.removeFile(fileList.next());
   }
 }
+
+/**
+ * Computes the Drive path for a file/folder using their Id
+ * @param {String} id The drive destination Id whose path needs to be computed
+ * @return {String} drivePath The corresponding drive path
+ */
+function getFullDrivePath(id, isFile) {
+  var current;
+
+  if (isFile) {
+    current = DriveApp.getFileById(id);
+  } else {
+    current = DriveApp.getFolderById(id);
+  }
+
+  var folders = [];
+  var parentIterator = current.getParents();
+
+  while (parentIterator.hasNext()) {
+    var parent = parentIterator.next();
+    folders.push(parent.getName());
+    parentIterator = parent.getParents();
+  }
+
+  var drivePath = folders.reverse().join('/') + '/' + current.getName();
+  return drivePath;
+}
+
+/**
+ * File System Type Enumeration
+ */
+var FileSystemType = {
+  WINDOWS: 'windows',
+  UNIX: 'unix',
+};
+
+/**
+ * @todo Write tests for validation
+ * @body Tests will be written in the directory API module as these are more closely related to that module.
+ */
+// Regex expressions to validate absolute paths
+var windowsPathRegExp = /^[\w]\:(\\|(\\[^<>\\/:"\|\?\*]+)+)\\?$/;
+var unixPathRegExp = /^(\/[^<>\\/:"\|\?\*]+)*\/?$/;
+
+/**
+ * Helper function to validate path
```

Comment:
Can you please add more details in the comments.. Mention that this is drive path you are referring to.
Just saying path might confuse the reader.

Reply:
Fixed